apps/explorer-e2e/src/support/pages/base-page.js

```diff
@@ -0,0 +1,79 @@
+export default class BasePage {
+  transactionsUrl = '/txs';
+  blocksUrl = '/blocks';
+  partiesUrl = '/parties';
+  assetsUrl = '/assets';
+  genesisUrl = '/genesis';
+  governanceUrl = '/governance';
+  marketsUrl = '/markets';
+  networkParametersUrl = '/network-parameters';
+  validatorsUrl = '/validators';
+  blockExplorerHeader = 'explorer-header';
+  searchField = 'search-input';
+
+  navigateToTxs() {
+    cy.get(`a[href='${this.transactionsUrl}']`).click();
+  }
+
+  navigateToBlocks() {
+    cy.get(`a[href='${this.blocksUrl}']`).click();
+  }
+
+  navigateToParties() {
+    cy.get(`a[href='${this.partiesUrl}']`).click();
+  }
+
+  navigateToAssets() {
+    cy.get(`a[href*='${this.assetsUrl}']`).click();
+  }
+
+  navigateToGenesis() {
+    cy.get(`a[href='${this.genesisUrl}']`).click();
+  }
+
+  navigateToGovernance() {
+    cy.get(`a[href='${this.governanceUrl}']`).click();
+  }
+
+  navigateToMarkets() {
+    cy.get(`a[href='${this.marketsUrl}']`).click();
+  }
+
+  navigateToNetworkParameters() {
+    cy.get(`a[href='${this.networkParametersUrl}']`).click();
+  }
+
+  navigateToValidators() {
+    cy.get(`a[href='${this.validatorsUrl}']`).click();
+  }
+
+  search(searchText) {
```

Comment:
should update the nming to be more explicit 'i. searchForTransaction , searchForMarket, searchForBlock ...

Reply:
In future iterations the inherited pages will have more specific methods for searching. Currently AFAIK the search doesn't work properly so this isn't being used at all yet.